.changelog/3973.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: Converted PB100 and PB104 validations to the new validation format.
+  type: internal
+pr_number: 3973
```

```suggestion
changes:
```
We don't report about new validations atm so remove this file.